Packs/RemoteAccess/Integrations/RemoteAccessv2/RemoteAccessv2.py

```diff
@@ -44,7 +44,7 @@ def perform_copy_command(ssh_client: SSHClient, file_path: str, destination_path
                 with tempfile.TemporaryDirectory() as temp_dir:
                     scp_client.get(file_path, f'{temp_dir}/{destination_path}')
                     with open(f'{temp_dir}/{destination_path}', 'rb') as f:
-                        return f.read()
+                        return str(f.read())
```

Comment:
better to change the return type of the function to `byte`, I'm not sure if it will always behave correctly if we cast it to string.

Reply:
done